msi/GitChooseEnvironmentDlg.wxs

```diff
@@ -3,15 +3,17 @@
   <Fragment>
     <UI>
       <Dialog Id="GitChooseEnvironmentDlg" Width="370" Height="270" Title="!(loc.GitChooseEnvironmentDlg_Title)" X="50" Y="50">
-        <Control Type="RadioButtonGroup" Property="CLIENVIRONMENT" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
-          <RadioButtonGroup Property="CLIENVIRONMENT">
+        <Control Type="RadioButtonGroup" Property="UI_CliEnvironment" Id="RadioButtonGroup" Width="310" Height="55" X="30" Y="87">
+          <RadioButtonGroup Property="UI_CliEnvironment">
```

Comment:
This change indicates that my previous hunch was right: it is better to introduce UI as well as logic for each new feature in the same commit (as opposed to introducing the complete, dysfunctional UI in one commit, and then successively adding code that handles those UI options).